File: Functions/Python/func.py
def say_hello():
    print("Hello, World!")

def add(a:int, b:int):
    return a + b

subtract = lambda a, b: a - b

def main():
    say_hello()
    print("5 + 2 =", add(5, 2))
    print(f"5 + 2 = {subtract(5, 2)}")

main()

File: Functions/C++/func.cpp
#include <iostream>

// Void Function
void say_hello() {
    std::cout << "Hello, World!" << std::endl;
}

// Return type function
int add(int a, int b) {
    return a + b;
}

// Lambda Function
auto subtract = [](int a, int b) -> int {
    return a - b;
};

int main(){
    say_hello();
    
    int sum = add(5, 2);
    int difference = subtract(5, 2);

    std::cout << "The sum of 5 and 2 is: " << sum << std::endl;
    std::cout << "The difference of 5 and 2 is: " << difference << std::endl; 

    return 0;
}